test/commands.test.ts

```diff
@@ -275,17 +277,19 @@ suite('Command Handler', () => {
         });
 
         test('errors if the server is not stopped', async () => {
+            sandbox.stub(vscode.window, 'showWarningMessage').resolves('Yes');
             statusStub.returns(ServerState.STARTED);
 
             try {
                 await handler.removeServer(serverState);
                 expect.fail();
             } catch (err) {
-                expect(err).equals('Please stop the server before removing it.');
+                expect(err).not.to.be.empty;
```

Comment:
Note that this clause will catch the expect.fail as assertion error as well. So the error variable is unlikely to ever be empty.

Reply:
@jrichter1 ok, didnt notice, but you're completely right. Nevertheless I dont want to assert specific error messages given that this will false positive whenever you want to change error messages (which is completely pointless).

Reply:
Then assert the error is not an instance of AssertionError.